assets/js/components/MiniChart.js

```diff
@@ -57,9 +62,10 @@ function MiniChart( { index, percent } ) {
 		width: 28,
 	};
 
+	const percent = +( change * 100 ).toFixed( 1 );
```

Comment:
This statement is used just to get a percentage number with 1 fraction digit. This will be consumed by google chats library. I don't think we need to produce a translated string here.